iosHackthonApp/Modules/Features/Login/Sources/Login/LoginView.swift

```diff
@@ -9,96 +9,93 @@
 import SwiftUI
 import shared
 import Components
-import Strings
```

Comment:
This file is a bit of a pain to see what I changed because me and Raynelle seem to have different indentation settings.
In here I removed the Strings import and moved all the strings references to the viewModel.
I also renamed the viewModel as there was only one in this file so didn't need to be specific about the name

Reply:
Might be good for us all to sync up our indentation style for the project.

Reply:
Definitely